README.md

```diff
@@ -19,6 +19,26 @@ permissions to run Coder.
 - Kubernetes Resosurces: checks that the cluster has the required
 resource types available to run Coder.
 
+## Installation
+
+You can manually download the latest [release](https://github.com/cdr/coder-doctor/releases):
+
+1. Click a release and download the tar file for your operating system
+   (ex: coder-doctor-linux-amd64.tar.gz)
+2. (Optional, recommended) Download the `checksums.txt` for the desired
+   release and verify the `sha256` checksum of the file you downloaded.
+3. Extract the `coder-doctor` binary.
+4. Move the `coder-doctor` binary to somewhere in your `$PATH`.
+
+Alternatively, use the below one-liner, replacing `$VERSION`, `$OSTYPE`
+(one of `darwin`, `linux`, `windows`) and `$ARCH` (one of `amd64`, `arm64`)
+as required:
+
+```bash
```

Comment:
As a general suggestion for shell commands, I think it's better to use `shell-session` and show example output, because it helps users know whether things are working correctly and they are seeing expected output